graph/resolver/discussion.resolvers.go

```diff
@@ -34,6 +35,22 @@ func (r *discussionResolver) Posts(ctx context.Context, obj *model.Discussion) (
 	return posts, nil
 }
 
+func (r *discussionResolver) PostsConnection(ctx context.Context, obj *model.Discussion, after *string) (*model.PostsConnection, error) {
+	/* Hardcode the number of posts per page. This can be changed to be settable by the client in the query. */
+	limit := 50
```

Comment:
Can be set to any other number you prefer. I assumed that 50 posts per page was a reasonable amount.

Reply:
50 sounds reasonable to me as well. Can you move this into a const at the top of this file please?